src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -134,7 +136,6 @@ module Iostat = struct
     (* Now read the values out of dev_values_map for devices for which we have data *)
     Listext.List.filter_map (fun dev ->
         if not (Hashtbl.mem dev_values_map dev) then (
-          D.debug "No iostat results for %s" dev;
           None
         ) else
           let values = Hashtbl.find dev_values_map dev in
```

Comment:
Think you can remove the parentheses now, and even put `if not [...] None` on one line.